Vocable/Features/Presets/Pagination/PaginationCollectionViewCell.swift

```diff
@@ -7,20 +7,36 @@
 //
 
 import UIKit
+import Combine
 
 class PaginationCollectionViewCell: VocableCollectionViewCell {
     
-    @IBOutlet weak var paginationLabel: UILabel!
+    let paginationLabel = UILabel(frame: .zero)
     
     var paginationDirection: UIPageViewController.NavigationDirection = .forward {
         didSet {
             updatePaginationLabel()
         }
     }
     
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+    
+    private func commonInit() {
+        addSubview(paginationLabel)
+        paginationLabel.translatesAutoresizingMaskIntoConstraints = false
+        paginationLabel.leftAnchor.constraint(equalTo: layoutMarginsGuide.leftAnchor).isActive = true
+        paginationLabel.rightAnchor.constraint(equalTo: layoutMarginsGuide.rightAnchor).isActive = true
+        paginationLabel.topAnchor.constraint(equalTo: layoutMarginsGuide.topAnchor).isActive = true
+        paginationLabel.bottomAnchor.constraint(equalTo: layoutMarginsGuide.bottomAnchor).isActive = true
```

Comment:
You don't need to make this change but you can add these constraints to a layout array that is active:
 
```suggestion
NSLayoutConstraint.activate([
         paginationLabel.leftAnchor.constraint(equalTo: layoutMarginsGuide.leftAnchor),
         paginationLabel.rightAnchor.constraint(equalTo: layoutMarginsGuide.rightAnchor),
         paginationLabel.topAnchor.constraint(equalTo: layoutMarginsGuide.topAnchor),
         paginationLabel.bottomAnchor.constraint(equalTo: layoutMarginsGuide.bottomAnchor)
])
```